.release-notes/4280.md

```diff
@@ -0,0 +1,5 @@
+## Introduction of empty Ranges
+
+Previously, a `Range` was considered infinite if either 1) the step is 0, or any of min, max, or step were NaN, +Inf or -Inf, or 2) if no progress could be made from min to max due to the sign of the step. This classification was a possible source of bugs including "hanging actors" when a `Range` would otherwise be reasonably expected to be **empty** -- producing no values. Through the RFC process the larger Pony community decided of the agreed-upon new mutually exclusive classification for what constitutes finite, infinite, and empty ranges.
```

Comment:
I don't think it is important that this came through the RFC process at least as part of the release notes. If you can bring more information about what changed into the release notes, that would be good with perhaps an "additional information available here" type link to the text of the RFC.

Reply:
I don't think this paragraph will mean anything to someone who hasn't read the RFC.

I'd like to see a large portion of the motivation section from the RFC written to the past tense and included as the bulk of the release notes to explain why a change was made and then the additional expansion of examples as noted below.

Reply:
I took a crack at rewriting to keep it short, but increase the amount of "why we made the change" included in the release notes. Take a look when you can and tell me what you think. I was struggling with how to not simply repeat the entire motivation section with all its additional details that would, in my opinion, lead to as much old/outdated code as new/working code without very clear delineation.